perf/terraform/modules/ci/files/cleanup.yml

```diff
@@ -13,7 +13,7 @@ Resources:
         Variables:
           REGIONS: '["us-west-2", "us-east-1"]'
           TAGS: '{"Project":"perf", "Name":"node"}'
-          MAX_AGE_MINUTES: '30'
+          MAX_AGE_MINUTES: '120'
```

Comment:
Second

Reply:
This YAML is only used for local testing. I just proposed moving the test files to a separate directory to make the two sets more distinct - https://github.com/libp2p/test-plans/pull/216